src/electron/main.js

```diff
@@ -189,9 +195,17 @@ app.whenReady().then(async () => {
 
   ipc.on('gitify:icon-idle', () => {
     if (!mb.tray.isDestroyed()) {
-      mb.tray.setImage(
-        updateAvailableMenuItem.visible ? idleUpdateAvailableIcon : idleIcon,
-      );
+      if (process.platform === 'linux') {
```

Comment:
Not 100% certain that all linux distros should use the white icon set 🤷‍♂️

Reply:
Yeah isn't it more about dark vs light mode than linux vs other OS's?

Reply:
I'll update this PR to make it a user-controlled system setting. That way it's more versatile and agnostic of os/platform

Reply:
Updated logic pushed to use a user controlled system setting